packages/jaeger-ui/src/model/transform-trace-data.tsx

```diff
@@ -32,6 +33,24 @@ function deduplicateTags(spanTags: Array<KeyValuePair>) {
   return { tags, warnings };
 }
 
+export function orderTags(spanTags: Array<KeyValuePair>, topPrefixes: Array<string>) {
+  const orderedTags: Array<KeyValuePair> = [...spanTags];
+  orderedTags.sort((a, b) => {
```

Comment:
I think this should be case insensitive, so below this I would add:
```
const aKey = a.key.toLowerCase();
const bKey = b.key.toLowerCase();
```
and then use `aKey` and `bKey` in place of `a.key` and `b.key`

and above `tree.walk` on line 111 I would add:
```
const prefixes = getConfigValue('topTagPrefixes');
const topTagPrefixes = prefixes && prefixes.map(p => p.toLowerCase())`;
```

Reply:
I think that `toLowerCase` mapping can be done inside `orderTags`. It's a bit of performance loss but makes code more testable.